indico/web/templates/_clipboard_input.html

```diff
@@ -1,8 +1,9 @@
 {%- macro clipboard_input(value, name='') -%}
-    <div class="i-has-action i-has-action-clip">
-        <input name="{{ name }}" type="text" value="{{ value }}" readonly>{#
-        #}<button type="button"
-         class="i-button icon-clipboard" title="Click to copy the URL"></button>
+    {% set id = "clipboard-" + uuid() %}
+    <div class="i-has-action">
+        <input id="{{ id }}" name="{{ name }}" type="text" value="{{ value }}" readonly>{#
```

Comment:
instead of the awful wrapping here you can put `{#--#}` in a new line:

```
<input id="{{ id }}" name="{{ name }}" type="text" value="{{ value }}" readonly>
{#--#}
<button ...
```